src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +22,14 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        Optional<Movie> movieOptional = movieDao.get(id);
+        return movieOptional.orElseThrow((Supplier<RuntimeException>) ()
+                -> new DataProcessingException("could not retrieve"
+                + " movie with id: " + id, null));
```

Comment:
```suggestion
        return movieDao.get(id).orElseThrow((Supplier<RuntimeException>) ()
                -> new DataProcessingException("could not retrieve"
                + " movie with id: " + id, null));
```
no need to create unnecessary variables